src/Payments.sol

```diff
@@ -0,0 +1,612 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Write down all the `getter` APIs.

Reply:
Try to make it as easy as possible to access basic state without an Indexer.

Reply:
Strike a balance between on-chain state and off-chain compute than an RPC provider needs to do to serve Getter requests.

Reply:
Storing the keys of a Hasmap in an a set if you need iteration. Do it only when needed.

Reply:
Added license and made the changes.